smokes/e2e/settings.scenarios.coffee

```diff
@@ -0,0 +1,80 @@
+# coffee script
+# test goal: checks the capability to navigate in a dedicated build
+# to use previous and next link
+
+builderPage = require('./builder.coffee')
+waterfallPage = require('./waterfall.coffee')
+settingsPage = require('./settings.coffee')
+
+describe('', () ->
+    builder = null
+    waterfall = null
+    settings = null
+
+    beforeEach(() ->
+        builder = new builderPage('runtests', 'force')
+        waterfall = new waterfallPage('runtests')
+        settings =  new settingsPage('runtests')
+        builder.goDefault()
+    )
+
+    scallingVar = '10'
+    describe 'manage settings', () ->
+        it 'should navigate to the setting, change the scalling factor and check it', () ->
+            settings.goSettings()
+            settings.changeScallingFactor(scallingVar)
+            waterfall.go()
+            settings.goSettings().then () ->
+                browser.sleep(1000)
```

Comment:
why a sleep? I dont thing there is a need. 

I dont think the then() is needed either